src/main/java/com/nhuqt/codefellowship/models/Post.java

```diff
@@ -0,0 +1,42 @@
+package com.nhuqt.codefellowship.models;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+
+@Entity
+public class Post {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  long id;
+  String body;
+  String createdAt;
+  @ManyToOne
+  ApplicationUser owner;
+
+  public ApplicationUser getOwner() {
+    return owner;
+  }
+
+  public Post(){}
```

Comment:
You generally want to keep your constructor functions together (any function you overload should be kept together so it's easy to see that they're overloaded.